### ISSUE TITLE
Add CI for PRs

### ISSUE THREAD

**Issue by Fleker** [googlesamples/assistant-sdk-cpp#58]
Add CI support within GitHub to auto-respond to PRs

**Comment by mpoullet** [comment]
For this you can have a look at my 3 branches:
- Ubuntu 18.04 on [AppVeyor](https://github.com/mpoullet/assistant-sdk-cpp/tree/appveyor): [appveyor.yml](https://github.com/mpoullet/assistant-sdk-cpp/blob/appveyor/appveyor.yml)
- Ubuntu 16.04 on [CircleCI](https://github.com/mpoullet/assistant-sdk-cpp/tree/circleci): [.circleci/config.yml](https://github.com/mpoullet/assistant-sdk-cpp/blob/circleci/.circleci/config.yml)
- Ubuntu 14.04 on [TravisCI](https://github.com/mpoullet/assistant-sdk-cpp/tree/travis): [.travis.yml](https://github.com/mpoullet/assistant-sdk-cpp/blob/travis/.travis.yml)

Only someone with commit rights to the original repository can set CI up on these CI services though.

**Comment by Fleker** [comment]
Yeah I'm playing around with [Google Cloud Build](https://cloud.google.com/cloud-build/) right now. Still trying to figure it out, but it could be pretty useful if it works.

**Comment by Fleker** [comment]
I'm getting it to pass now.
